src/main/java/com/minecolonies/coremod/colony/buildings/workerbuildings/BuildingGlassblower.java

```diff
@@ -132,39 +118,6 @@ public void onColonyTick(@NotNull final IColony colony)
         super.onColonyTick(colony);
     }
 
-    @Override
-    public void checkForWorkerSpecificRecipes()
-    {
-        super.checkForWorkerSpecificRecipes();
-
-        final List<IToken<?>> tokens = new ArrayList<>();
-        for (final Item item : Tags.Items.SAND.getAllElements())
-        {
-            final ItemStack stack = new ItemStack(item);
-            final ItemStack output = FurnaceRecipes.getInstance().getSmeltingResult(stack);
-            if (Tags.Items.GLASS.contains(output.getItem()))
```

Comment:
So, it's probably not critical, but this change means that any moded sands that are included in this now no longer get recipes.

Reply:
Yes, we were discussing that in #discussion on Discord.  The conclusion was that nobody could think of any mods in which "(item tagged as sand) => (item tagged as glass)" were standard furnace smelting recipes.  But if one does come up, a modpack could include a compat recipe datapack.

Reply:
Ok, I've been away all day, I'll go catch up there.

Reply:
Also, Ray's other idea is that once crafting modules are added, the glassblower could have both a crafting and smelting module and then you'd have to teach it which glass recipe you want to use (based on glass product tag) rather than these being inherent crafter custom recipes.  This avoids most of the potential compat issues, as long as things are tagged appropriately.  (And it wouldn't have to care about sand, just glass.)

The first edition of the JEI PR won't be quite enough to get it there, but Soon™.